src/sgame/sg_bot_util.cpp

```diff
@@ -221,44 +221,31 @@ Scoring functions for logic
 =======================
 */
 
-botEntityAndDistance_t BotGetClosestBuildingAmongTypes(
-		gentity_t *self, const std::initializer_list<buildable_t> buildables )
+botEntityAndDistance_t BotGetHealTarget( const gentity_t *self )
 {
-	botEntityAndDistance_t best_choice = { nullptr, 1.0e30f };
-	for ( buildable_t buildable : buildables )
+	if ( G_Team( self ) == TEAM_HUMANS )
 	{
-		botEntityAndDistance_t candidate =
-			self->botMind->closestBuildings[ buildable ];
-		if ( candidate.ent && candidate.distance < best_choice.distance )
-		{
-			best_choice = candidate;
-		}
+		// may be null with near infinite distance
+		return self->botMind->closestBuildings[BA_H_MEDISTAT];
 	}
-	return best_choice;
-}
 
-const gentity_t *BotGetHealTarget( gentity_t *self )
-{
-	if ( G_Team(self) == TEAM_HUMANS )
-	{
-		return self->botMind->closestBuildings[BA_H_MEDISTAT].ent;
-	}
+	static constexpr glm::vec3 up( 0.0f, 0.0f, 1.0f );
 
-	// Aliens
-	if ( self->botMind->closestBuildings[BA_A_BOOSTER].ent )
-	{
-		// powered booster
-		return self->botMind->closestBuildings[BA_A_BOOSTER].ent;
-	}
-	else
+	for ( buildable_t buildable : { BA_A_BOOSTER, BA_A_OVERMIND, BA_A_SPAWN, BA_A_LEECH })
```

Comment:
Is it intentional to always prefer e.g. overmind over spawn regardless of distance?

Reply:
Here, yes because in the previous version it caused trouble, but since the buildable types are more selected I'd say I could restore it.

I'm really unsure but this is the most conservative option, I guess